iroh/src/client/blobs.rs

```diff
@@ -1,5 +1,50 @@
 //! API for blobs management.
-
+//!
+//! The main entry point is the [`Client`].
+//!
+//! ## Interacting with the local blob store
+//!
+//! ### Importing data
+//!
+//! There are several ways to import data into the local blob store:
+//!
+//! - [`add_bytes`](Client::add_bytes)
```

Comment:
Stylewise IMO "imports in-memory data" reads nicer than "allows importing" (this applies to a few more sentences as well)